astropy/nddata/utils.py

```diff
@@ -456,3 +469,267 @@ def block_replicate(data, block_size, conserve_sum=True):
         data = data / float(np.prod(block_size))
 
     return data
+
+
+class Cutout(object):
+    def __init__(self, data, position, shape, wcs=None, mode='trim',
+                 fill_value=np.nan):
+        """
+        Create a cutout object from a 2D array.
+
+        The returned object will contain the 2D cutout array and the
+        updated WCS, if a `~astropy.wcs.WCS` object is input.
```

Comment:
`cutout.data` is a view of the original large `data` array.  `cutout.wcs` is a copy of the original `wcs` with the updated `CRPIX` values.  I've updated the docs.